views/layouts/main.handlebars

```diff
@@ -13,6 +13,7 @@
 
     {{!-- main layout styling --}}
     <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/team.css"> <!--quick fix to load team.css.  The 'include other styling' portion doesn't seem to load this. -->
 
     {{!-- include other styling --}}
     {{#each css}}
```

Comment:
I think this is a location issue. Down here I think you can change this to: "../../static/css/{{this}}" and the location should be alright.

Reply:
or just "/static/css/{{this}}" actually